interfaces/automated-testing/scd/design/strategic-coordination-interoperability/nominal-test.plantuml

```diff
@@ -0,0 +1,28 @@
+@startuml
+center header Strategic Coordination Interoperability
+title Nominal test
```

Comment:
Suggestion to put a test number to this e.g. Nominal Test OPIN001 something like that

Reply:
OPIN001 is a reference used in the standard and it could be confusing. If you agree, I would suggest to introduce test indexes when we will add other use cases.